src/_includes/soft_mod_business_case_presentation_banner.html

```diff
@@ -0,0 +1,8 @@
+{% capture background_image %}/assets/custom/img/softmod/bg-business-case-presentation-template.jpg{% endcapture %}
+{% assign background_position = 'top left' %}
+{% capture description %}{% t software-modernisation.business-case-presentation-banner-text %}{% endcapture %}
+{% capture cta_href %}
+      https://info.codurance.com/en/case-for-change-questionnaire
+{% endcapture %}
+{% capture cta_text %}{% t software-modernisation.business-case-presentation-banner-cta-text %}{% endcapture %}
+{% include in_page_banner.html background_image=background_image background_position=background_position class="soft-mod-cheat-sheet-cta-banner" cta_href=cta_href cta_text=cta_text description=description split="right-heavy" %}
```

Comment:
I'm surprised that this one is not using a HubSpot CTA. Is that correct?

Reply:
Is this include being used at all?

Reply:
This will need a cta created in Hubspot and it will be used within a blog post by José once the deck that is to be downloaded is finally signed off and finished. 

We can then upload it to Hubspot and put this include in.